ERROR - 2025-10-15 18:44:54 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pagina2', 3306, '', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\turismoya\app\Controllers\ActividadController.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\ActividadController->index()
#8 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ActividadController))
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\turismoya\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\turismoya\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\turismoya\app\Controllers\ActividadController.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\ActividadController->index()
#7 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ActividadController))
#8 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-10-15 18:44:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: actividades]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `actividad`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ActividadController.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ActividadController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ActividadController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-15 19:35:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 19:35:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-15 19:35:27 --> mysqli_sql_exception: Table 'pagina2.usuario' doesn't exist in engine in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\turismoya\app\Models\pagprincipal.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\turismoya\app\Controllers\usuario.php(55): App\Models\pagprincipal->validarusuario('cacaca@gmail.co...', 'cacaca')
#8 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\usuario->iniciarSesion()
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-10-15 19:35:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pagina2.usuario' doesn't exist in engine
[Method: POST, Route: login/iniciar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`
WHERE `gmail` = :gmail:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\pagprincipal.php(18): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\usuario.php(55): App\Models\pagprincipal->validarusuario('cacaca@gmail.com', 'cacaca')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->iniciarSesion()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 19:35:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pagina2.usuario' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuario`
WHERE `gmail` = \'cacaca@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuario`
WHERE `gmail` = \'cacaca@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`
WHERE `gmail` = :gmail:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\pagprincipal.php(18): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\usuario.php(55): App\Models\pagprincipal->validarusuario('cacaca@gmail.com', 'cacaca')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->iniciarSesion()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 19:35:27 --> [Caused by] mysqli_sql_exception: Table 'pagina2.usuario' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuario`
WHERE `gmail` = \'cacaca@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuario`
WHERE `gmail` = \'cacaca@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuario`
WHERE `gmail` = \'cacaca@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`
WHERE `gmail` = :gmail:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\pagprincipal.php(18): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\usuario.php(55): App\Models\pagprincipal->validarusuario('cacaca@gmail.com', 'cacaca')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->iniciarSesion()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-15 19:47:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-15 19:48:30 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`pagina4`.`usuario`, CONSTRAINT `usuario_ibfk_1` FOREIGN KEY (`id_rol`) REFERENCES `rol` (`id_rol`)) in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\turismoya\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\turismoya\app\Models\usuariomodel.php(20): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\turismoya\app\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario(Array)
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\usuario->registro()
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
#11 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-15 19:48:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`pagina4`.`usuario`, CONSTRAINT `usuario_ibfk_1` FOREIGN KEY (`id_rol`) REFERENCES `rol` (`id_rol`))
[Method: POST, Route: registro/guardar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 19:48:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`pagina4`.`usuario`, CONSTRAINT `usuario_ibfk_1` FOREIGN KEY (`id_rol`) REFERENCES `rol` (`id_rol`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'celina\', \'maldonado\', \'celimaldonado@gmail.com\', \'$2y$10$J7MVImshtq0D43i.t/jgz.tFuWJpCAJ8UNi.RXUFCVMImD43MLM4u\', 2)')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'celina\', \'maldonado\', \'celimaldonado@gmail.com\', \'$2y$10$J7MVImshtq0D43i.t/jgz.tFuWJpCAJ8UNi.RXUFCVMImD43MLM4u\', 2)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 19:48:30 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`pagina4`.`usuario`, CONSTRAINT `usuario_ibfk_1` FOREIGN KEY (`id_rol`) REFERENCES `rol` (`id_rol`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'celina\', \'maldonado\', \'celimaldonado@gmail.com\', \'$2y$10$J7MVImshtq0D43i.t/jgz.tFuWJpCAJ8UNi.RXUFCVMImD43MLM4u\', 2)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'celina\', \'maldonado\', \'celimaldonado@gmail.com\', \'$2y$10$J7MVImshtq0D43i.t/jgz.tFuWJpCAJ8UNi.RXUFCVMImD43MLM4u\', 2)')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'celina\', \'maldonado\', \'celimaldonado@gmail.com\', \'$2y$10$J7MVImshtq0D43i.t/jgz.tFuWJpCAJ8UNi.RXUFCVMImD43MLM4u\', 2)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-15 19:52:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 19:52:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 19:52:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 19:52:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 19:53:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:33:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:33:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-15 20:36:24 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\turismoya\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\turismoya\app\Models\usuariomodel.php(20): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\turismoya\app\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario(Array)
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\usuario->registro()
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
#11 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-15 20:36:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: registro/guardar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:36:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$SZo.GQ1vDQ29xdalBn8LmOsbqY1gBeXMzbH6B1IMWrD2eY/5HcOwS\', 2)')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$SZo.GQ1vDQ29xdalBn8LmOsbqY1gBeXMzbH6B1IMWrD2eY/5HcOwS\', 2)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:36:24 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$SZo.GQ1vDQ29xdalBn8LmOsbqY1gBeXMzbH6B1IMWrD2eY/5HcOwS\', 2)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$SZo.GQ1vDQ29xdalBn8LmOsbqY1gBeXMzbH6B1IMWrD2eY/5HcOwS\', 2)')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$SZo.GQ1vDQ29xdalBn8LmOsbqY1gBeXMzbH6B1IMWrD2eY/5HcOwS\', 2)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-15 20:36:59 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\turismoya\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\turismoya\app\Models\usuariomodel.php(20): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\turismoya\app\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario(Array)
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\usuario->registro()
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
#11 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-15 20:36:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: registro/guardar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:36:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$A/8W3Dxcd/rkASVtmPKnkeN.f/JOKemisHovk6ihRddyVIyH/pvnm\', 2)')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$A/8W3Dxcd/rkASVtmPKnkeN.f/JOKemisHovk6ihRddyVIyH/pvnm\', 2)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:36:59 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$A/8W3Dxcd/rkASVtmPKnkeN.f/JOKemisHovk6ihRddyVIyH/pvnm\', 2)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$A/8W3Dxcd/rkASVtmPKnkeN.f/JOKemisHovk6ihRddyVIyH/pvnm\', 2)')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'turismoya\', \'turismoya\', \'turismoya@gmail.com\', \'$2y$10$A/8W3Dxcd/rkASVtmPKnkeN.f/JOKemisHovk6ihRddyVIyH/pvnm\', 2)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-15 20:37:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:37:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:37:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:37:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:37:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-15 20:37:43 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\turismoya\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\turismoya\app\Models\usuariomodel.php(20): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\turismoya\app\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario(Array)
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\usuario->registro()
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
#11 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-15 20:37:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: registro/guardar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:37:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'paula\', \'brondino\', \'paubrondino@gmail.com\', \'$2y$10$kqxW6s8Ph1NLpuFjyAwNwOQiFvZd7n56S7KeGVfbJW2TfSCKc0ig.\', 2)')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'paula\', \'brondino\', \'paubrondino@gmail.com\', \'$2y$10$kqxW6s8Ph1NLpuFjyAwNwOQiFvZd7n56S7KeGVfbJW2TfSCKc0ig.\', 2)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:37:43 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'paula\', \'brondino\', \'paubrondino@gmail.com\', \'$2y$10$kqxW6s8Ph1NLpuFjyAwNwOQiFvZd7n56S7KeGVfbJW2TfSCKc0ig.\', 2)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'paula\', \'brondino\', \'paubrondino@gmail.com\', \'$2y$10$kqxW6s8Ph1NLpuFjyAwNwOQiFvZd7n56S7KeGVfbJW2TfSCKc0ig.\', 2)')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (\'paula\', \'brondino\', \'paubrondino@gmail.com\', \'$2y$10$kqxW6s8Ph1NLpuFjyAwNwOQiFvZd7n56S7KeGVfbJW2TfSCKc0ig.\', 2)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`nombre`, `apellido`, `gmail`, `clave`, `id_rol`) VALUES (:nombre:, :apellido:, :gmail:, :clave:, :id_rol:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\usuariomodel.php(20): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\usuario.php(32): App\Models\usuariomodel->registrarUsuario([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario->registro()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-15 20:38:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:38:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:38:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:38:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:39:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:40:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-15 20:41:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-15 20:41:57 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\turismoya\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `ac...', 0)
#1 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ac...')
#2 C:\xampp\htdocs\turismoya\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ac...')
#3 C:\xampp\htdocs\turismoya\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ac...', Array, false)
#4 C:\xampp\htdocs\turismoya\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\turismoya\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\turismoya\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\turismoya\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\turismoya\app\Controllers\Actividad.php(46): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(933): App\Controllers\Actividad->guardar()
#10 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Actividad))
#11 C:\xampp\htdocs\turismoya\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\turismoya\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\turismoya\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\turismoya\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-15 20:41:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'field list'
[Method: POST, Route: actividad/guardar]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (:nombre:, :precio:, :descripcion:, :disponibilidad:, :imagen:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Actividad.php(46): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Actividad->guardar()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Actividad))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:41:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (\'Avistaje de aves\', \'10000\', \'Observar el vuelo de aves es una opción de ecoturismo para disfrutar de la naturaleza en pleno Buenos Aires.\\r\\n\', 3, \'1760560917_4cbf046922dbab522a3b.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (\'Avistaje de aves\', \'10000\', \'Observar el vuelo de aves es una opción de ecoturismo para disfrutar de la naturaleza en pleno Buenos Aires.\\r\\n\', 3, \'1760560917_4cbf046922dbab522a3b.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (:nombre:, :precio:, :descripcion:, :disponibilidad:, :imagen:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Actividad.php(46): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Actividad->guardar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Actividad))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-15 20:41:57 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (\'Avistaje de aves\', \'10000\', \'Observar el vuelo de aves es una opción de ecoturismo para disfrutar de la naturaleza en pleno Buenos Aires.\\r\\n\', 3, \'1760560917_4cbf046922dbab522a3b.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (\'Avistaje de aves\', \'10000\', \'Observar el vuelo de aves es una opción de ecoturismo para disfrutar de la naturaleza en pleno Buenos Aires.\\r\\n\', 3, \'1760560917_4cbf046922dbab522a3b.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (\'Avistaje de aves\', \'10000\', \'Observar el vuelo de aves es una opción de ecoturismo para disfrutar de la naturaleza en pleno Buenos Aires.\\r\\n\', 3, \'1760560917_4cbf046922dbab522a3b.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `actividad` (`nombre`, `precio`, `descripcion`, `disponibilidad`, `imagen`) VALUES (:nombre:, :precio:, :descripcion:, :disponibilidad:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Actividad.php(46): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Actividad->guardar()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Actividad))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
